code/game/turfs/simulated/walls.dm

```diff
@@ -209,14 +213,34 @@
 			projectile_reflection(Proj)
 			return PROJECTILE_CONTINUE // complete projectile permutation
 
-	//cut some projectile damage here and not in projectile.dm, becouse we need not to all things what are using get_str_dam() becomes thin and weak.
+	//cut some projectile damage here and not in projectile.dm, because we need not to all things what are using get_str_dam() becomes thin and weak.
```

Comment:
this entire comment makes no sense. As long as you're correcting it, maybe rewrite it

Reply:
I have no idea what it even means I just force of habit corrected the one misspelling I noticed. Someone who actually knows what this is trying to say should rewrite it.